### DISCUSSION TITLE
Ground Items Fill

### DISCUSSION THREAD

**Discussion by SamuraiPR** [runelite/runelite#15308] [Ideas]
**Is your feature request related to a problem? Please describe.**
With the current release of beams I decided to try it, however being a 90 degrees camera player I barely benefit from it.

**Describe the solution you'd like**
It would be nice to have an option to fill the ground item square highlight instead of just a square outline, this will make it more eye catching in my case. Opacity adjustment is a must to avoid clicking empty area.

**Describe alternatives you've considered**
1.) Allowing to change the ground item square outline thickness.

2.) If the beam could have a strong glowing effect to become more eye catching it might work better, but if this alternative has notable increase in GPU usage, then it is not ideal for my scenario.

**Additional context**
The following image would be the ideal result:

![FILL](https://user-images.githubusercontent.com/61925549/126529081-ad57db7d-bbc1-4939-a7b2-6bb0ae8ccf4f.png)

**Comment by Nightfirecat** [comment]
The new loot beam model should be much more visible from basically any angle, and should address this nicely.
